notebooks/SMAP/smap_rasterio_issues.ipynb

```diff
@@ -0,0 +1,2339 @@
+{
```

Comment:
<p>Update from the on-prem URL to cloud: <a href="https://data.nsidc.earthdatacloud.nasa.gov/nsidc-cumulus-prod-protected/SMAP/SPL4SMGP/007/2015/03/31/SMAP_L4_SM_gph_20150331T013000_Vv7032_001.h5" target="_blank">https://data.nsidc.earthdatacloud.nasa.gov/nsidc-cumulus-prod-protected/SMAP/SPL4SMGP/007/2015/03/31/SMAP_L4_SM_gph_20150331T013000_Vv7032_001.h5</a></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/nsidc/NSIDC-Data-Tutorials/pull/91/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
I wonder if we leave this rasterio access pattern out of the tutorials for the moment.  As I recall I was playing around with regridding.